src/main/java/it/gov/pagopa/payhub/activities/activity/customdebtposition/DebtPositionSynchronizeFineActivity.java

```diff
@@ -0,0 +1,27 @@
+package it.gov.pagopa.payhub.activities.activity.customdebtposition;
```

Comment:
I would prefer a nested package inside debtposition, but specialized for fine use case
```suggestion
package it.gov.pagopa.payhub.activities.activity.debtposition.custom.fine;
```